src/main/java/com/twilio/http/HttpClient.java

```diff
@@ -50,6 +53,10 @@ public Response reliableRequest(final Request request, final int[] retryCodes, i
             // Decrement retries
             retries--;
         }
+
+        last_request = request;
+        last_response = response;
```

Comment:
These should be `lastRequest` and `lastResponse` to follow Java naming conventions.